packages/wasm/test/test.js

```diff
@@ -51,6 +51,7 @@ test('fetching WASM from separate file', async (t) => {
 
   global.result = null;
   global.t = t;
+  // eslint-disable-next-line global-require, import/no-dynamic-require
```

Comment:
does it not work to use `import` here instead of `require`?

Reply:
If `import` is used, `t.plan(3)` needs to be changed to `t.plan(2)`.
Not sure if there will be any unexpected consequences as a result of that.